src/core/ui.js

```diff
@@ -34,6 +34,21 @@ function ariaDecorate(elem, ariaMap) {
   }, elem);
 }
 
+const observer = new IntersectionObserver(
+  entries => {
+    entries.forEach(entry => {
+      if (entry.intersectionRatio > 0) {
+        entry.target.setAttribute("visibility", "hidden");
+      } else {
+        entry.target.setAttribute("visibility", "visible");
+      }
+    });
+  },
+  { root: document.querySelector("#logo") }
```

Comment:
Note that specs can have multiple logos.  Logos have a “.logo” class.

Reply:
Ohh yes ! That just didn't crossed my mind.

Reply:
I wonder if the root shouldn’t be respecUI? The you observe() each logo?

Reply:
Yeah the root needs to be the logo with which we need to observe out button as in this case it needs to be the w3c logo... But since I thought it will be the only logo I assigned root to be the logo.. that's my fault here

Reply:
So here it will be the respec-ui as we will be observing with respect to the whole ui.